app/views/layouts/application.html.erb

```diff
@@ -60,16 +60,13 @@
       </div>
     </header>
     <div class="govuk-width-container">
-      <div class="govuk-phase-banner">
-        <p class="govuk-phase-banner__content">
-          <strong class="govuk-tag govuk-phase-banner__content__tag">
-            alpha
-          </strong>
-          <span class="govuk-phase-banner__text">
-            This is a new service – your <%= ghwt_contact_mailto(label: "feedback", subject: "Feedback") %> will help us to improve it.
-          </span>
-        </p>
-      </div>
+      <%= render GovukComponent::PhaseBanner.new(phase: 'Alpha') do
+        if current_user.persisted? && (current_user.is_responsible_body_user? || current_user.is_school_user?)
+          "#{ t('user_feedback')} – #{ user_feedback_link }".html_safe
+        else
+          "This is a new service – your #{ ghwt_contact_mailto(label: "feedback", subject: "Feedback") } will help us to improve it.".html_safe
```

Comment:
cant every get the google form? why a different journey?

Reply:
We only want signed-in users to be able to use the survey otherwise we might have responses from other parties.